packages/block-editor/src/components/rich-text/format-toolbar-container.js

```diff
@@ -51,14 +55,14 @@ function InlineToolbar( { anchorRef } ) {
 	);
 }
 
-const FormatToolbarContainer = ( { inline, anchorRef, value } ) => {
+const FormatToolbarContainer = ( { inline, editableContentRef, value } ) => {
```

Comment:
Other notable change in this PR: I decided to rename `anchorRef` to `editableContentRef` in this file, as it is better representing what that variable is for.

It looks like the `FormatToolbarContainer` is not exported from the package anyway, so I think this change is ok.